3rdParty/V8/CMakeLists.txt

```diff
@@ -353,6 +353,9 @@ else ()
     set(V8_CFLAGS   "${V8_CFLAGS} -fno-delete-null-pointer-checks")
     set(V8_CXXFLAGS "${V8_CXXFLAGS} -fno-delete-null-pointer-checks")
   endif()
+  if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+      set(V8_CXXFLAGS "${V8_CXXFLAGS} -Wno-suggest-override -Wno-implicit-const-int-float-conversion -Wno-non-virtual-dtor -Wno-final-dtor-non-final-class -Wno-implicit-const-int-float-conversion")
+  endif()
 
```

Comment:
pls also set V8_CFLAGS - just to have this symetric.

Reply:
Done.